pydotorg/urls.py

```diff
@@ -1,71 +1,72 @@
-from django.conf.urls import include, url, handler404
```

Comment:
These were from @tjguk who did the initial job of working out how this should be structured. The new line 50 of this file is the important thing to retain.